lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -971,6 +971,67 @@ setReadOnly( Complex64Array.prototype, 'get', function get( idx ) {
 	return getComplex64( this._buffer, idx );
 });
 
+/**
+* Returns the first index at which a given element can be found.
+*
+* @name indexOf
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Complex64} searchElement - element to find
+* @param {integer} [searchFrom=0] - starting index (inclusive)
```

Comment:
```suggestion
* @param {integer} [fromIndex=0] - starting index (inclusive)
```

Matching the naming conventions in the MDN docs.